core/pom.xml

```diff
@@ -100,7 +100,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Good catch, @ewencp.

@josephadler: I'd love to use a tool for this if you already have one!